api/handler/api.go

```diff
@@ -28,11 +28,13 @@ type (
 		DefaultMaxAge      int
 		NotificatorEnabled bool
 		TLSEnabled         bool
+		CopiesNumber       uint32
 	}
 )
 
 // DefaultPolicy is a default policy of placing containers in NeoFS if it's not set at the request.
 const DefaultPolicy = "REP 3"
+const DefaultCopiesNumber uint32 = 0
```

Comment:
```suggestion
const (
    DefaultPolicy = "REP 3"
    DefaultCopiesNumber uint32 = 0
)
```

Also add comment for `DefaultCopiesNumber`